packages/http/src/resources/PendingTransactions.ts

```diff
@@ -7,17 +7,22 @@ type Context = Account
 
 export default class PendingTransactions {
   private client!: Client
+
   private context?: Context
 
   constructor(client: Client, context?: Context) {
     this.client = client
     this.context = context
   }
 
-  async get(hash: string): Promise<PendingTransaction> {
+  async get(hash: string): Promise<PendingTransaction[]> {
     const url = `/pending_transactions/${hash}`
     const { data: { data } } = await this.client.get(url)
-    return new PendingTransaction(data)
+    const pendingTransactions: PendingTransaction[] = []
+    data.forEach((txn: HTTPPendingTransactionObject) => {
+      pendingTransactions.push(new PendingTransaction(txn))
+    })
+    return pendingTransactions
```

Comment:
should this return a resource list instead of an array? not sure if they're ever going to be cursor-paginated, but at least it would be consistent with all the other methods that return lists of items. and you could call `take` on it etc

Reply:
The reason I didnt do that is because it wasnt cursor paginated, so its always returning the entire array. Do you think we need resource list for that?

Reply:
The list should be fairly small, as it really only have multiple when a transactions fails and hopefully we aren't failing more than like 10 times haha

Reply:
I think we should still return it as a resource list even if it's not paginated now because those still act like an array, and also have the `take` helper. that way we're always returning a consistent resource

Reply:
Ok let me update it

Reply:
Updated now